lib/core-integration/src/Test/Integration/Scenario/API/Shelley/TransactionsNew.hs

```diff
@@ -202,16 +255,45 @@ spec = describe "NEW_SHELLEY_TRANSACTIONS" $ do
             , expectField (#fee . #getQuantity) (`shouldBe` expectedFee)
             , expectField (#coinSelection . #withdrawals) (`shouldSatisfy` (not . null))
             ]
-        -- TODO: sign and submit tx,
-        --       check that reward account is 0,
-        --       make sure wallet balance is increased by withdrawalAmt - fee
+
+        -- TODO check that reward account is 0 (??)
```

Comment:
Piotr, do you know what this means/how I could check it is true?

Reply:
Oh, I believe it means just to assert that reward balance on the rewardWallet is 0 after withdrawal.

```
expectField  (#balance . #reward . #getQuantity) (`shouldBe` 0)
```
(or smth)